1003-button-design-guidance.md

```diff
@@ -0,0 +1,66 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-15
+updated: 2024-10-22
+---
+
+# Button Design Guidance
+
+The [button](https://clarity.design/documentation/button) component allows users to trigger actions or events with a single click.
+
+## Guidance
+
+Clarity supports solid, outline and flat buttons. These types are provided to give visual distinction between the priority or hierarchy of the buttons in the application.
+
+### Button Types
+
+- **Must** use solid buttons for the primary action that the application is suggesting that the user take.
+- **Must** use outline buttons to indicate a secondary action that complements a primary action.
+- **Should** use outline buttons to reduce visual noise when there are many actions of equal importance on the page.
+- **Should** use flat buttons when a user is expected to take an action within the page.
+- **Should** use a link (or anchor element) in place of a button to navigate the user to a different page.
```

Comment:
```suggestion
- Design teams **should** use a link (or anchor element) instead of a button to navigate the user to a different page, as this distinction helps clarify the action.
```